### PR TITLE
wb-disable: Adding option to allow some plugins on basic mode

### PR DESCRIPTION
**Pull request by Garneauma**
Adding option to allow a few plugins to run on basic mode:
- Data-ajax
- Data-JSON
- JSON manager
- Postback

Also adding wb-disable documentation.

### CODE REVIEW

File: src/plugins/wb-disable/wb-disable-fr.hbs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+{
+"title": "HTML simplifié",
+"language": "fr",
+"category": "Plugins",
```
Comment (duboisp) [inline src/plugins/wb-disable/wb-disable-fr.hbs:5]: Voici le correctif pour l'erreur relevé par le link checker de notre CI script

```suggestion
"category": "Plugiciels",
```

**Review by duboisp — DISMISSED** [review]
I locally and manually tested every wb-disable-allowed init and it worked as expected.

It's great to see the introduction of our GCWeb index.json-ld here which will allow us to use it in the future.

There is an CI error that need to be fixed, see my inline comment.

Great work on this.

File: src/plugins/wb-disable/wb-disable-fr.hbs (hunk 1 above)
Comment (Garneauma) [reply src/plugins/wb-disable/wb-disable-fr.hbs]: Unfortunate that `grunt dist` or `grunt test` don't notice that error. It is fixed now!

**Review by duboisp — DISMISSED** [review]

**Review by GormFrank — DISMISSED** [review]
In this PR, we must take into consideration the fact that a domino effect is possible here, since both `data-ajax` as well as `wb-data-json` can trigger init events after dynamic data is loaded, assuming they have WET plugins within that data; resulting in a potential Basic HTML mode that isn't so basic after all.

**Comment by Garneauma** [comment]
> In this PR, we must take into consideration the fact that a domino effect is possible here, since both `data-ajax` as well as `wb-data-json` can trigger init events after dynamic data is loaded, assuming they have WET plugins within that data; resulting in a potential Basic HTML mode that isn't so basic after all.

Data loaded via data-ajax are now taken into consideration. Users must add the class `.wb-disable-allow` to elements loaded in order for them to be initialized in basic mode.

**Review by duboisp — CHANGES_REQUESTED** [review]
I tried locally,  and I do confirm:
* Data ajax is initiated in basic mode
   * In my previous review I tested all init method and it worked. Here I only tested the init with `[data-ajax-replace]`
* json-manager component in the ajaxed contetn is initiated if activated do run in basic mode
* data-json component in the ajaxed content is initiated if activated do run in basic mode
  * I only tested the init with : `data-wb-json`

But the wb-postback plugin don't start.

TODO:
* [x] Ensure the wb-postback plugin can start in basic mode

**Comment by Garneauma** [comment]
> I tried locally, and I do confirm:
> 
> * Data ajax is initiated in basic mode
>   
>   * In my previous review I tested all init method and it worked. Here I only tested the init with `[data-ajax-replace]`
> * json-manager component in the ajaxed contetn is initiated if activated do run in basic mode
> * data-json component in the ajaxed content is initiated if activated do run in basic mode
>   
>   * I only tested the init with : `data-wb-json`
> 
> But the wb-postback plugin don't start.
> 
> TODO:
> 
> * [x]  Ensure the wb-postback plugin can start in basic mode

Fixed wb-postback.

Apparently, using `.wb-postback` as the selector for the init event does not work. It only works when using `[data-wb-postback]`. I can't figure out why by looking at the code.

**Review by duboisp — APPROVED** [review]
Reviewed and tested locally,

This is a minor change - Add a new basic init feature for data-ajax, data-json, json-manager and postback.

**Pull request merged**